web/wp-content/themes/dctx-v2/inc/customizer/class-text-editor-custom-control.php

```diff
@@ -0,0 +1,79 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-dgnypy.php, but found phpcs-scan-dgNYpY.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).